src/components/Layout/KFooter.vue

```diff
@@ -1,12 +1,14 @@
 <template>
   <div class="footer bg-neutral font-primary">
-    <div class="footer-max-width row generic-padding justify-around">
-      <div class="row generic-padding justify-around">
+    <div class="footer-max-width row generic-padding justify-between">
+      <div/>
```

Comment:
Do we need these self-closing divs? Maybe there is a quasar layout/flexbox/column type thing that works?

Reply:
Yeah, I somehow couldn't get it done otherwise. The "justify-around" option was doing the trick on desktop, but would also add margins on mobile (which was why it was indented so weirdly). I will have a look at the content options. Maybe those help...

...I actually thought of redoing the footer completely somewhen anyways though since I'm still not too happy with it.

Reply:
Ok, lets leave it like that for now then if it will be redone in not so long :)